### PR TITLE
Documentation Changes

### PR DESCRIPTION
**Pull request by emcarru**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Contributing to the Flynt Starter Theme
+
+Thank you for taking the time to contribute! Read the [Contributing To Flynt guidelines here](https://github.com/flyntwp/guidelines/blob/master/CONTRIBUTING.md).
```
Comment (Qakulukiam) [inline .github/CONTRIBUTING.md]: "to" should be lowercase

File: README.md [hunk 2]
```diff
@@ -180,10 +180,11 @@ The main people in charge of this repo are:
 
 ## Contribute
 
-To contribute, please use github [issues](https://github.com/flyntwp/flynt-starter-theme/issues). Pull requests are accepted.
+To contribute, please use GitHub [issues](https://github.com/flyntwp/flynt-starter-theme/issues). Pull requests are accepted. Please also take a moment to read the [contributing guidelines](https://github.com/flyntwp/guidelines/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/flyntwp/guidelines/blob/master/CODE_OF_CONDUCT.md).
```
Comment (Qakulukiam) [inline README.md]: it's called Code of Conduct in the file, but lowercase code of conduct here. Which one do we go with?

File: README.md [hunk 3]
```diff
@@ -180,10 +180,11 @@ The main people in charge of this repo are:
...
 
 If editing the README, please conform to the [standard-readme](https://github.com/RichardLitt/standard-readme) specification.
 
+
```
Comment (Qakulukiam) [inline README.md]: why the extra space?

File: README.md (hunk 2 above)
Comment (emcarru) [inline README.md]: I will use Code of Conduct, that's more consistent.

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
 # Contributing to the Flynt Starter Theme
 
-Thank you for taking the time to contribute! Read the [Contributing To Flynt guidelines here](https://github.com/flyntwp/guidelines/blob/master/CONTRIBUTING.md).
+Thank you for taking the time to contribute! Read the [Contributing to Flynt guidelines here](https://github.com/flyntwp/guidelines/blob/master/CONTRIBUTING.md).
```
Comment (Qakulukiam) [inline .github/CONTRIBUTING.md]: Actually, let's call this "Flynt Contributing Guidelines" (as in the README.md)

**Pull request merged**